components/AppRouterMigrationComponents/AppFooter.tsx

```diff
@@ -0,0 +1,324 @@
+'use client';
+
+import Link from 'next/link';
+import React from 'react';
+import { BsDiscord } from 'react-icons/bs';
+import GithubIconSvg from '../../public/svg/github-icon.svg';
+import LinkedInIconSvg from '../../public/svg/linkedin-icon.svg';
+import XIconSvg from '../../public/svg/x-icon.svg';
+import YoutubeIconSvg from '../../public/svg/youtube-icon.svg';
+import '../../styles/tailwind.css';
+import { TinaIcon } from '../logo';
+import { DynamicLink } from '../ui';
+
+const footerNav = [
```

Comment:
is there a PBI to migrate this hardcoded content into TinaCMS?
add a comment with a link to it here

Reply:
Done, commented in both locations